jwst/fits_generator/create_dms_data.py

```diff
@@ -3,7 +3,7 @@
 import os
 import numpy
 from astropy.io import fits as pyfits
-import jwst.fits_generator
+from .. import fits_generator
 
```

Comment:
I fixed this by "from .. import *". It has many functions in the `__init__`. Fix coming shortly.